src/app/sitemap.js

```diff
@@ -0,0 +1,10 @@
+export default function sitemap() {
+  return [
+    {
+      url: 'https://eddiehub.org',
+      lastModified: new Date(),
+      changeFrequency: 'monthly',
+      priority: 1,
+    },
+  ]
+}
```

Comment:
```suggestion
}

```

Reply:
Hello Krish,

Please can you help me understand why we need a space, I am new to this your support will be greatly appreciated!

Thanks!

Reply:
@ctrl07 We have a newline at the end in all of our files, so it is to maintain consistency

Reply:
Thanks! I will keep it in mind.

Reply:
I completed it from the suggestions itself. Not able to figure out if I got it correctly.